Sources/VaporSecurityHeaders/Configurations/ReferrerPolicyConfiguration.swift

```diff
@@ -14,13 +14,17 @@ public struct ReferrerPolicyConfiguration: SecurityHeaderConfiguration {
         case unsafeUrl = "unsafe-url"
     }
 
-    private let option: Options
+    private let headerValue: String
 
-    public init(_ option: Options) {
-        self.option = option
+    public init(_ directive: Directive) {
+        headerValue = directive.rawValue
+    }
+
+    public init(_ directives: [Directive]) {
+        headerValue = directives.map({ $0.rawValue }).joined(separator: ", ")
```

Comment:
Let's move the String conversion to `setHeader()`
```suggestion
        self.directives = directives
```